realm/realm-library/src/androidTest/java/io/realm/TestHelper.java

```diff
@@ -355,22 +355,27 @@ public int read() throws IOException {
         }
     }
 
+    @Deprecated
```

Comment:
Need @deprecated javadoc with explanation of what to use now that this is deprecated. Example of this: http://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html#javadoc_tag